pkg/piecestore/psserver/server.go

```diff
@@ -52,7 +52,7 @@ func (c Config) Run(ctx context.Context, server *provider.Provider) (err error)
 
 	ctx, cancel := context.WithCancel(ctx)
 
-	s, err := Initialize(ctx, c, server.Identity().Key)
+	s, err := Initialize(ctx, c, server.Identity().Key, zap.S())
```

Comment:
change to a named logger
`zap.L().Named(fmt.Sprintf("storage node (%s)", server.Identity().ID))`

Reply:
SugaredLogger can log formatted messages so we'll keep using that